Composite/Composite.Cs.Tests/Composites/MarkedCompositeEmptyTests.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Linq;
+
+using Xunit;
+
+using Composite;
+using Composite.Cs.Tests.Infrastructure;
+
+namespace Composite.Cs.Tests.Composites
+{
+    public class MarkedCompositeEmptyTests
+    {
+        [Fact]
+        public void LazyOutputTest(){
+            var result = MarkedComposite.Empty<string, Simple>("mark").AsEnumerable().AllowTake(0);
+
+            Assert.Throws<InvalidOperationException>(() =>
+            {
+                result.ToArray();
+            });
+        }
+    }
+}
```

Comment:
It seems like the test does not test Marked Composite functionality and it covers the case which already tested in AllowTakeTests.TrivialParametersTest().

Reply:
True. Looks like the test is useless. Will replace it with a test to check for the generation of the composite.